commit ffbd1a29a9b238716ebde02948c3d22aaaab4bda
Author: Joachim Nilsson <troglobit@gmail.com>
Date:   Sun May 15 22:23:45 2016 +0200

    Fix C-a, C-space, C-e regression from --disable-cscope patch
    
    This fixes a regression introduced in 658bad7, the --disable-cscope
    patch.  Where C-a, C-e, C-space (set mark), et al stopped working.
    
    Signed-off-by: Joachim Nilsson <troglobit@gmail.com>

diff --git a/keymap.c b/keymap.c
index 7630bcd..e80cca7 100644
--- a/keymap.c
+++ b/keymap.c
@@ -421,6 +421,10 @@ static struct KEYMAPE (8) fundmap = {
 		{
 			CCHR('@'), CCHR('G'), fund_at, (KEYMAP *) & ccmap
 		},
+#else
+		{
+			CCHR('@'), CCHR('G'), fund_at, NULL
+		},
 #endif
 		{
 			CCHR('H'), CCHR('H'), fund_h, (KEYMAP *) & helpmap
